src/api/app/views/webui2/webui/project/_monitor_control.html.haml

```diff
@@ -7,8 +7,9 @@
         %button.btn.btn-outline-secondary.dropdown-toggle{ data: { toggle: :dropdown }, type: :button }
           %span.caret
           Status
-        .dropdown-menu
-          %button.btn.btn-link.monitor-no-filter-link{ type: :button } No filter
+        .dropdown-menu.keep-open
+          %button.btn.btn-link.monitor-filter-link{ type: :button } All
+          %button.btn.btn-link.monitor-no-filter-link{ type: :button } Any
```

Comment:
Should the string instead of "Any" be "None"? And later in two more places, the same.